apps/nextjs-website/src/components/atoms/AlertPart/AlertPart.tsx

```diff
@@ -5,8 +5,8 @@ import Alert, { AlertColor } from '@mui/material/Alert';
 export type AlertPartProps = {
   alertStyle?: SxProps;
   severity: AlertColor;
-  text: string;
-  title: string;
+  text?: string;
+  title?: string;
```

Comment:
Is this right? Why should be allowed to create an Alert without text and title? 🤔

Reply:
Fixed. Only text should be optional.